src/repository/crl.rs

```diff
@@ -488,6 +488,15 @@ where
 pub struct RevokedCertificates(Captured);
 
 impl RevokedCertificates {
+    /// Create an empty RevokedCertificates for use in SignedMessage
```

Comment:
The doc attribute should have a first line summarizing the purpose – probably just something like ‘Creates an empty list.’ here.

Reply:
ack, also removed the remaining - this is probably not the place to write prose